### PR TITLE
Fix #13

### LINKED ISSUE #13: ~> Conflicts with the Swift STL
Ah, the totally foreseeable consequences of keeping every operator, no matter how private, at global scope.

### PR DESCRIPTION
**Pull request by pthariensflame**
Renamed:
- `~>` to `~~>`
- `>~` to `>~~`
- `<~` to `<~~`
- `~<` to `~~<`

I also took the liberty of removing the “FIXME” and updating the tutorial appropriately.

### CODE REVIEW

File: Aquifer/Simple.swift [hunk 1]
```diff
@@ -62,45 +62,43 @@ public func for_<UO, UI, DI, DO, NI, NO, FR>(p: Proxy<UO, UI, DI, DO, FR>, _ f:
...
 
-// FIXME: The Swift STL thinks it's OK to leave all operators, even private ones, at global public
-// scope.
-// infix operator ~> {
-// associativity right
-// precedence 130
-// }
+infix operator ~~> {
+associativity right
+precedence 130
```
Comment (CodaFi) [inline Aquifer/Simple.swift]: Can you indent all of these?

File: Aquifer/Simple.swift [hunk 2]
```diff
@@ -62,45 +62,43 @@ public func for_<UO, UI, DI, DO, NI, NO, FR>(p: Proxy<UO, UI, DI, DO, FR>, _ f:
...
+}
 
 /// Into | Composes two loops to yield one large loop.
-public func ~> <IS, UO, UI, DI, DO, NI, NO, FR>(f: IS -> Proxy<UO, UI, DI, DO, FR>, g: DO -> Proxy<UO, UI, NI, NO, DI>) -> IS -> Proxy<UO, UI, NI, NO, FR> {
+public func ~~> <IS, UO, UI, DI, DO, NI, NO, FR>(f: IS -> Proxy<UO, UI, DI, DO, FR>, g: DO -> Proxy<UO, UI, NI, NO, DI>) -> IS -> Proxy<UO, UI, NI, NO, FR> {
```
Comment (CodaFi) [inline Aquifer/Simple.swift:73]: These should have the note about their pipes equivalent.

File: Aquifer/Simple.swift (hunk 1 above)
Comment (pthariensflame) [inline Aquifer/Simple.swift]: How would you like them indented?

File: Aquifer/Simple.swift (hunk 2 above)
Comment (pthariensflame) [inline Aquifer/Simple.swift:73]: Ok!

File: Aquifer/Simple.swift (hunk 1 above)
Comment (CodaFi) [inline Aquifer/Simple.swift]: As in Operadics

**Comment by pthariensflame** [comment]
How does it look now?

**Comment by CodaFi** [comment]
:+1:

**Comment by CodaFi** [comment]
#ShipIt

**Pull request merged**